docs/data/tree-view/accessibility/accessibility.md

```diff
@@ -61,6 +61,11 @@ When a single-select tree receives focus:
 - If none of the items are selected when the tree receives focus, focus is set on the first item.
 - If an item is selected before the tree receives focus, focus is set on the selected item.
 
+|                         Keys | Description                                                  |
```

Comment:
@noraleonte Is there a reason not to add those here?
It allows to link to a table dedicated to single selection from the selection page.

Reply:
Hmm, that's a good point 🤔 But I think there would be some duplicated information in this case. `Enter` is already described in the first table (line 40), and `space` in the table for multi-select trees. How about we move the description of the `Space` key to the main table, and leave these other two sections to handle only specific interactions? Does that make sense?

Reply:
Space does not toggle, it only selects (see #12654)
So I don't think we can have this one shared.

Reply:
Oh, right. In that case, let's go with your version. 👌